src/web/router.js

```diff
@@ -109,6 +109,7 @@ router.get('/discrepancies/search', auth.secured, discrepancies.search)
 router.post('/discrepancies/search', auth.secured, discrepancies.getDiscrepancyReport)
 router.post('/discrepancies/resolve/:id', auth.secured, discrepancies.resolveDiscrepancy)
 
+router.get('/stripe/balance', auth.secured, stripe.balance)
```

Comment:
Any secured route will require a session to be present and valid (stored in a HTTP secure cookie). This can currently only be established by carrying out GitHub oauth and having the GitHub account be part of team with permissions.

For this particular resource I think it's OK to lean on IP allow-listing + security groups/ IAM to be confident that only processes that should be allowed to invoke this are, so it should be okay to keep this route open (removing auth secured).

FYI the reason this is working locally is because your local `.env` file has the environment variable `AUTH_GITHUB_ENABLED` set to `false`, if you changed that to true (like in any actual environment) the route would `403` requiring an appropriate session.